http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoMetricsMonitor.scala

```diff
@@ -156,8 +156,6 @@ class LeoMetricsMonitor[F[_]](config: LeoMetricsMonitorConfig,
                 authHeader = Authorization(Credentials.Token(AuthScheme.Bearer, token))
                 relayPath = Uri.unsafeFromString(baseUri.asString) / app.appName.value
                 isUp <- serviceName match {
-                  case ServiceName("wds") =>
```

Comment:
I'm not familiar with this part of the code so feel free to blow me off, but it looks like this part is just all apps and not specifically cbas, so should wds stay here?

Reply:
Aaron and I had a similar discussion above ^ I think this still lists the wds within cbas which will no longer there and we will need to create a new separate metric for standalone WDS in the future.